extension.js

```diff
@@ -57,9 +61,14 @@ const SpotifyLabel = new Lang.Class({
 		this._refresh();
 	},
 
-	// Update left padding of this.buttonText according to new value set in settings
+	// Update padding of this.buttonText according to new value set in settings
 	_leftPaddingChanged: function() {
-		this.buttonText.set_style("padding-left: " + this.settings.get_int('left-padding') + "px;");
+		this.buttonText.set_style("padding-left: " + this.settings.get_int('left-padding') + "px; "
```

Comment:
I'd like the `padding-right` setting to get it's own line, as in:
`this.buttonText.set_style("padding-right: " + this.settings.get_int('right-padding') + "px;");`
even if it means a slight bit of code repetition in favour of a bit more readabe code.

Reply:
The method name `set_style` implied it overwrites the current style, indicating this would get rid of the left-padding? Or does `set_style` just append it to the current style?

Reply:
Oh. I'm actually not sure, but I had (naively?) hoped it would be smart and only set the correct setting - but a quick look at the docs seems to support your idea. Keep this as it is in that case, good catch.

Reply:
I just tested it and it does indeed replace the current style. Also, I missed setting the style on initialization to include the right-padding in the style, I'll add this and implement the other thing in `prefs.js` and request review :)